Makefile

```diff
@@ -535,7 +535,7 @@ $(OBJDIR)/petsc-% : examples/petsc/%.c examples/petsc/libutils.a.PHONY $(libceed
 	  PETSC_DIR="$(abspath $(PETSC_DIR))" OPT="$(OPT)" $*
 	cp examples/petsc/$* $@
 
-$(OBJDIR)/fluids-% : examples/fluids/%.c $(libceed) $(ceed.pc) | $$(@D)/.DIR
+$(OBJDIR)/fluids-% : examples/fluids/%.c examples/fluids/src/*.c examples/fluids/*.h examples/fluids/problems/*.c examples/fluids/qfunctions/*.h $(libceed) $(ceed.pc) | $$(@D)/.DIR
```

Comment:
Oops! Sorry I missed this. What was the problem it was creating that I never noticed?

Reply:
I like to build and run from the top level. So I edit a source file in the library *or* in the example, and it should always build cleanly with minimal work. Without these listed prerequisites, editing source files other than `navierstokes.c` would not call the sub-directory make.

Reply:
I see! Thanks for clarifying!